docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -6,26 +6,70 @@ title: prometheus.scrape
 
 # prometheus.scrape
 
-`prometheus.scrape` configures a metrics scraping job for a given set of
+`prometheus.scrape` configures a Prometheus scraping job for a given set of
 `targets`. The scraped metrics are forwarded to the list of receivers passed in
 `forward_to`.
 
 Multiple `prometheus.scrape` components can be specified by giving them
 different labels.
 
+## Scraping behavior
+The `prometheus.scrape` component borrows the scraping behavior of Prometheus.
+Prometheus, and by extend this component, uses a pull model for scraping
+metrics from a given set of _targets_.
+Each scrape target is defined as a set of _labels_ i.e. key-value pairs. The
```

Comment:
```suggestion
Each scrape target is defined as a set of _labels_, i.e., key-value pairs. The
```

Reply:
Apparently it's recommended to avoid e.g. and i.e. because people tend to mix them up: 

https://developers.google.com/style/abbreviations?hl=en#dont-use

Reply:
@rfratto, @tpaschalis , yeah, sorry, I used to always use "for example" rather than "i.e." but at DataRobot, we used "i.e." and "e.g" because the data scientists liked them (and even used them in the right way).  I wasn't sure about our software engineer users but you're right -- we shouldn't use them.  We can go back and update the ones we didn't change at some point. The Google style guide is pretty adamant about this.  ;-)

Reply:
I've removed these from both PRs. We'll have to find some new obscure latin abbreviations to confuse users with 😅